newIDE/app/src/UI/Theme/Theme/Table.css

```diff
@@ -0,0 +1,140 @@
+:root{
+  --gd-default-font: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
+  Oxygen-Sans, Ubuntu, 'Helvetica Neue', Helvetica, sans-serif,
+  'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+color: black;
+
+/*defaulttheme*/
+}
+:root {
+  --white: white;
+  --black: black;
+  --table-font-size: 13px;
+  --table-font-border-bottom-default-theme: 1px solid #e0e0e0;
+  --table-odd-row: #fafafa;
+  --table-Selected-Row: #4c92ff;
+  --table-Header-Column-font-weight: 600;
+  --table-Header-Column-font-size: 12px;
+  --table-Header-Column-color: #9e9e9e;
+  --table-Header-Column-text-transform: none;
+  --table-Column-white-space: nowrap;
+  --table-Column-overflow: hidden;
+  --table-Column-text-overflow: ellipsis;
+}
+/*darktheme*/
```

Comment:
Each theme should have its own "Table.css" file (or a single css file, up to you), that contains just variables for the colors. Things like whitespace, margin, padding, overflow, font-size, etc... must be in a single CSS file used by all themes.